docker/Dockerfile

```diff
@@ -1,26 +1,20 @@
-FROM crystallang/crystal:1.4.1-alpine AS builder
+FROM crystallang/crystal:1.8.2-alpine AS builder
 RUN apk add --no-cache sqlite-static yaml-static
-
```

Comment:
Why did you remove all these lines?

Reply:
Same in docker/Dockerfile.arm64, there is no need to remove all the existing spaces

Reply:
@xbdmHQ please revert the changes about the spaces removed

Reply:
> @xbdmHQ please revert the changes about the spaces removed

Done

Reply:
There are still spaces to modify :smiley:, look at the Dockerfile: https://github.com/iv-org/invidious/pull/4103/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddc

Line 23 removed - please revert
Line 36 unnecessary additional space - please revert

Note: I'm fine with line 2 with the additional space it's ok.

Reply:
> There are still spaces to modify 😃, look at the Dockerfile: https://github.com/iv-org/invidious/pull/4103/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddc
> 
> Line 23 removed - please revert Line 36 unnecessary additional space - please revert
> 
> Note: I'm fine with line 2 with the additional space it's ok.

Fixed